### ISSUE TITLE
Pulsing skeletons are distracing for the no-JS experience

### ISSUE THREAD

**Issue by knowler** [npmx-dev/npmx.dev#1129]
We shouldn’t show the pulsing skeletons loading UI unless we know JS is working as for those browsing without JS, they are quite annoying (and could [pose accessibility issues](https://www.w3.org/WAI/WCAG22/quickref/#pause-stop-hide) since they’re animations that cannot be stopped). The most effective way of checking for that is having a class or attribute on the `<html>` element and removing it as early as possible. Then we can create a variant which just prepends `.no-js` to the beginning of a complex selector, then use `no-js:hidden` (or something) for stuff that needs to be hidden.

Avoid the new `scripting` media feature as that doesn’t cover JS being blocked through web extensions (such as content blockers) which is likely more common as a method to block scripts than the browser setting.

**Comment by danielroe** [comment]
checking, with some of my newer changes, I think it's only the 'Weekly downloads' skeleton that renders without JS...
